### ISSUE TITLE
Two equal strings with different language tags considered equal

### ISSUE THREAD

**Issue by mommi84** [digitalbazaar/jsonld.js#293]
Given the following triples,

```turtle
<http://example.com> <http://example.com/label> "test"@en .
<http://example.com> <http://example.com/label> "test"@fr .
```

only the first one is being saved to JSON-ld using `fromRDF()`.

```jsonld
[ { '@id': 'http://example.com',
    'http://example.com/label': [ { '@value': 'test', '@language': 'en' } ] } ]
```

Tested with **jsonld@1.5.2**.

**Comment by davidlehn** [comment]
Yes, this is a bug.  It's a typo in the N-Quads parser in rdf-canonize.  I'll add some tests, get the fix double checked, and get releases out later today.  Thanks!

If you need a quick fix, I think the issue is in NQuads.js, _compareTriples:
```diff
-  if(t1.termType !== 'Literal') {
+  if(t1.object.termType !== 'Literal') {
```

@dlongley Does this fix look right?

**Comment by davidlehn** [comment]
Fixed in 1.5.3.
